src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -343,6 +350,12 @@ protected void validateAdmission(Admission admission, boolean insert) throws OHS
 		}
 		if (admission.getDiseaseIn() == null) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.admission.diagnosisincannotbeempty.msg")));
+		} else {
+			List<Disease> diseases = diseaseManager.getDiseaseIpdIn().stream().filter(dis -> dis == admission.getDiseaseIn()).collect(Collectors.toList());
+			if ( diseases.size() > 0) {
```

Comment:
```suggestion
			if (diseases.size() > 0) {
```

Reply:
The blank space appears to be still there.

Reply:
anyway, the error should be thrown if the diagnosis is NOT found in the stream, hence when `diseases.isEmpty()`, correct?